heron/io/dlog/src/java/org/apache/heron/dlog/DLInputStream.java

```diff
@@ -6,9 +6,9 @@
  * to you under the Apache License, Version 2.0 (the
  * "License"); you may not use this file except in compliance
  * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
+ * <p>
```

Comment:
Please revert this change, to be consistent to all other JAVA files.

Reply:
Ok, I saw the Travis CI build failed because of this, will revert accordingly.

Reply:
thx